lib/web_ui/lib/src/engine/window.dart

```diff
@@ -114,6 +115,10 @@ base class EngineFlutterView implements ui.FlutterView {
   late final PlatformViewMessageHandler platformViewMessageHandler =
       PlatformViewMessageHandler(platformViewsContainer: dom.platformViewsHost);
 
+  // TODO(goderbauer): Provide API to configure constraints. See also TODO in "render".
```

Comment:
This will happen through the new JS Embedder API, when calling addView on an app.